src/BloomExe/CollectionTab/LibraryModel.cs

```diff
@@ -392,6 +392,31 @@ private static byte[] GetBookReplacedWithTemplate(string bookFile)
 			if (match.Success)
 				text = text.Substring(0, match.Index) + text.Substring(match.Index + match.Length);
 
+			// BL-2476: Readers made from BloomPacks should have the formatting dialog disabled
+			regex = new Regex("\\s*<meta\\s+name=(['\\\"])pageTemplateSource\\1 content=(['\\\"])(Leveled|Decodable) Reader\\2>(</meta>)? *");
+			match = regex.Match(text);
+			if (match.Success)
+			{
+				// has the suppressFormatting meta tag been added already?
+				var regexSuppress = new Regex("\\s*<meta\\s+name=(['\\\"])suppressFormatting\\1 content=(['\\\"])(.*)\\2>(</meta>)? *");
+				var matchSuppress = regexSuppress.Match(text);
+				if (matchSuppress.Success)
+				{
+					// the meta tag already exists, make sure the value is "true"
+					if (matchSuppress.Groups[3].Value.ToLower() != "true")
+					{
+						text = text.Substring(0, matchSuppress.Groups[3].Index) + "true"
+							+  text.Substring(matchSuppress.Groups[3].Index + matchSuppress.Groups[3].Length);
+					}
+				}
+				else
+				{
+					// the meta tag has not been added, add it now
+					text = text.Insert(match.Index + match.Length,
+						"\r\n    <meta name=\"suppressFormatting\" content=\"true\"></meta>");
+				}
+			}
+
```

Comment:
I think we may have unit tests for bloompack creation. If so, it would be nice to cover these new cases.